sensor_labels:
  - imu_id: 00B4738B
    label: imu_head